src/sql/workbench/services/query/common/queryRunner.ts

```diff
@@ -495,11 +496,19 @@ export default class QueryRunner extends Disposable {
 	 * @param includeHeaders [Optional]: Should column headers be included in the copy selection
 	 */
 	async copyResults(selections: Slick.Range[], batchId: number, resultId: number, includeHeaders?: boolean): Promise<azdata.CopyResultsRequestResult> {
+		let copyInBackend = false;
```

Comment:
The checks are being made from the sqlops-dataprotocolclient package before the copy request is made to STS: https://github.com/microsoft/sqlops-dataprotocolclient/pull/101/files

Reply:
@lewis-sanchez 
I believe this check should happen here instead of in the dataprotocolclient, which is only responsible for communication, IIUC the actual implementation should live in STS.
see my comment in the dataprotocolclient PR: https://github.com/microsoft/sqlops-dataprotocolclient/pull/101/files#r1733520645